src/main/java/org/example/onlinebookstore/controller/AuthenticationController.java

```diff
@@ -17,7 +17,7 @@
         description = "Endpoints for managing user authentication and registration")
 @RequiredArgsConstructor
 @RestController
-@RequestMapping("/auth")
+@RequestMapping("/api/auth")
```

Comment:
add `server.servlet.context-path=/api` to app properties

```suggestion
@RequestMapping("/auth")
```